### PR TITLE
user dynamically loaded as intials as avatar

### PR DESCRIPTION
**Pull request by teddapoet**
- Modified user and event models, added extra entries in the participants table for testing, changed event controller to add user to the json file
- Users are now dynamically loaded with initials as avatar in the footer of event cards
To do : 
- Modal styling
- Add participant counter function ?

### CODE REVIEW

File: app/javascript/events.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 let cachedEvents = JSON.parse(localStorage.getItem("cachedEvents")) || [];
+console.log("Cached Events from localStorage:", cachedEvents);
```
Comment (AndyZzzZzzZzz) [inline app/javascript/events.js]: Is this for debugging purposes?

File: app/javascript/events.js [hunk 2]
```diff
@@ -144,7 +145,7 @@ function initializeEvents() {
     const loadingMessage = document.getElementById("loading-message");
     
     // Show the loading message and clear previous content
-    loadingMessage.style.display = "block";
+    // loadingMessage.style.display = "block";
```
Comment (AndyZzzZzzZzz) [inline app/javascript/events.js]: Are we removing the loading messages? If so, there's no reason to keep it as a comment

File: app/javascript/modal.js [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-    elmnt.style.left = (elmnt.offsetLeft - pos1) + "px";
-  }
+//   function elementDrag(e) {
+//     e = e || window.event;
+//     e.preventDefault();
+//     // Calculate the new cursor position
+//     pos1 = pos3 - e.clientX;
+//     pos2 = pos4 - e.clientY;
+//     pos3 = e.clientX;
+//     pos4 = e.clientY;
```
Comment (AndyZzzZzzZzz) [inline app/javascript/modal.js]: A bit confused on changes happening in this file. Seems like you added a bunch of comments?

File: app/javascript/events.js (hunk 2 above)
Comment (teddapoet) [reply app/javascript/events.js]: i dont remember commenting out this, maybe a mistake, i can add it in again

File: app/javascript/events.js (hunk 1 above)
Comment (teddapoet) [reply app/javascript/events.js]: yes i had to include it to check whether the json file is loading user in the console, it wont have any effect on the front end

File: app/javascript/modal.js (hunk 3 above)
Comment (teddapoet) [reply app/javascript/modal.js]: this was for the pop up stuffs, i didnt know i want to delete it or not

File: app/javascript/events.js (hunk 1 above)
Comment (AndyZzzZzzZzz) [reply app/javascript/events.js]: If there are no bugs in the final version of the code, rmeove any debug statements. We don't wanna stuff unrelated stuff in codebase

File: app/javascript/events.js (hunk 2 above)
Comment (AndyZzzZzzZzz) [reply app/javascript/events.js]: Decide on whether we want this functionality or not and make corresponding changes

File: app/javascript/modal.js (hunk 3 above)
Comment (AndyZzzZzzZzz) [reply app/javascript/modal.js]: sounds good

**Pull request merged**